environmental-values.csl

```diff
@@ -289,6 +290,7 @@
             <text macro="collection-title" prefix="(Serie " suffix=") "/>
             <text macro="locators" suffix=": "/>
             <text variable="page"/>
+            <text variable="DOI" prefix="https://doi.org/"/>
```

Comment:
You only want to add DOIs for articles, not e.g. for reports or book chapters?

Reply:
It is not clear from the instructions. Probably best to allow DOIs there too. I am not sure how. One thing that is missing is the instruction to provide DOI on a new line. I tried implementing that but without success. Could you show me how?